packages/top-app-bar/README.md

```diff
@@ -48,7 +48,8 @@ If you decide to use a React Component please see [Integrating with Components](
 
 > NOTE: `actionItems` prop is expecting an array of elements.
 
-Please see [MDC Web Top App Bar Readme](https://github.com/material-components/material-components-web/tree/master/packages/mdc-top-app-bar) for more information.
+> NOTE: If you use [Material Icon's](../material-icon/README.md) as Action Items or Navigation Icons, you will get the Ripple for free. Using custom elements will require you to
+> wrap the custom element with the [`withRipple` HOC](../ripple/README.md).
```

Comment:
nope nope, If you use custom elements then you have to implement the hasRipple prop.

Reply:
They will need the `withRipple` HOC. Even if they have `hasRipple` prop, it won't be attached to anything. In the example below, `hasRipple` won't do anything:

```js
const Icon = ({hasRipple}) => (
  <i className='icon-class-name' hasRipple >menu</i>
);
```